rest-api/pom.xml

```diff
@@ -56,6 +56,12 @@
             <version>${workflow.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-input-step</artifactId>
+            <version>2.0</version>
+            <optional>true</optional>
```

Comment:
How this configuration behaves exactly? Jenkins plugin manager will try to install it only _if possible_, if so, what's _if possible_ in this context?

Sorry for my questions, it's not clear to me how `optional` works here.